tests/csm/rest/test_query_deployment.py

```diff
@@ -0,0 +1,113 @@
+                                                K8S_SCRIPTS_PATH)
+        assert resp[0], resp[1]
+        self.log.info("Cleanup: Cluster destroyed successfully")
+
+        self.log.info("Cleanup: Setting prerequisite")
+        self.deploy_lc_obj.execute_prereq_cortx(self.csm_obj.master,
+                                                K8S_SCRIPTS_PATH,
+                                                K8S_PRE_DISK)
+
+        for node in self.csm_obj.worker_list:
+            self.deploy_lc_obj.execute_prereq_cortx(node, K8S_SCRIPTS_PATH,
+                                                    K8S_PRE_DISK)
+        self.log.info("Cleanup: Prerequisite set successfully")
+
+        self.log.info("Cleanup: Deploying the Cluster")
+        resp_cls = self.deploy_lc_obj.deploy_cluster(self.csm_obj.master,
+                                                     K8S_SCRIPTS_PATH)
+        assert resp_cls[0], resp_cls[1]
+        self.log.info("Cleanup: Cluster deployment successfully")
+
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+        time.sleep(self.update_seconds)
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+
+        self.log.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.poll_cluster_status(self.csm_obj.master)
+        assert resp[0], resp[1]
+        self.log.info("Cleanup: Cluster status checked successfully")
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-45675')
+    def test_45675(self):
+        """
+        Verify GET cluster topology with valid storage id
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Send GET request for fetching system topology"
+                      "without storage set id")
+        self.log.info("Get cluster id from system topology response")
+        get_topology = self.csm_obj.get_system_topology()
+        for cluster in get_topology["topology"]["cluster"]:
+            resp, result, err_msg = self.csm_obj.verify_storage_set(cluster_id = cluster['id'])
+            assert result, err_msg
+            self.log.info("Response : %s", resp)
+            self.log.info("Step 2: Send GET request for fetching system topology"
+                      "with storage set id")
+            #need to revisit
+            storage_sets = resp.json()["topology"]["cluster"][0]["storage_set"]
+            for storage_set_id in storage_sets:
+                self.log.info("Sending request for %s ", storage_set_id)
+                resp, result, err_msg = self.csm_obj.verify_storage_set(cluster_id = cluster['id'],
+                                                storage_set_id = storage_set_id['id'])
+                assert result, err_msg
+                self.log.info("Response : %s", resp)
+        self.log.info("##### Test ended -  %s #####", test_case_name)
```

Comment:
Add new line

Reply:
added

Reply:
No new line is added. Can u please check again.